app/views/layouts/application.html.erb

```diff
@@ -105,8 +105,30 @@
   </div>
 </div>
 <% unless internal_navigation? %>
+  <script>
+    function toggleFooterVisibility() {
+      const footer = document.querySelector("#footer");
+      if (!footer) {
+        return;
+      }
+      if (window.location.pathname === "/") {
+        footer.style.display = "none";
+      } else {
+        footer.style.display = "block";
+      }
+    }
+    // To ensure correct footer visibility even when opening a child page in a new tab,
+    // we add a DOMContentLoaded event listener, which calls toggleFooterVisibility
+    // after the entire page, including the footer, is fully loaded.
+    document.addEventListener("DOMContentLoaded", function () {
+      toggleFooterVisibility();
+    });
+    toggleFooterVisibility();
+  </script>
   <% cache("footer-and-signup-modal-#{user_signed_in?}-#{ForemInstance.deployed_at}-#{Settings::General.admin_action_taken_at&.rfc3339}", expires_in: 24.hours) do %>
-    <%= render "layouts/footer" %>
+    <div id="footer" style="display:none">
```

Comment:
I'd prefer using our "hidden" CSS class than an inline style, you can toggle it with `footer.classList.add` and `footer.classList.remove`

Also it may be better to apply styling to the footer in "layouts/footer" directly

Reply:
Thanks for this consistent change. Applied this in code.